### ISSUE TITLE
Add Windows support

### ISSUE THREAD

**Issue by mewim** [kuzudb/go-kuzu#12]

**Comment by Anwenya** [comment]
please！

**Comment by andreasdim** [comment]
Hello
Do you have this on the roadmap?
Best regards

**Comment by mewim** [comment]
> Hello Do you have this on the roadmap? Best regards

Yes this is on our roadmap, but since we are a small team and currently busy with other priorities, we cannot promise a specific timeframe on the completion of this feature. If you have a specific use case / project requirement blocked by this, please join our Discord channel at https://kuzudb.com/chat and we can discuss more about whether we need to prioritize this more.

**Comment by mewim** [comment]
Update: this is done. It was more straightforward than I expected.
